dot-net-sdk/EppoClient.cs

```diff
@@ -191,7 +196,19 @@ public class EppoClient
         return assignment;
     }
 
-    public static EppoClient Init(EppoClientConfig eppoClientConfig)
+    public static EppoClient InitClientMode(EppoClientConfig eppoClientConfig) =>
+        InitInner(
```

Comment:
My vote would be to refresh configurations right after this

Reply:
done in InitInner, explicitly for all deployment types